NSURLSessionCancellation/ViewController.swift

```diff
@@ -27,9 +27,9 @@ class ViewController: UIViewController {
 
         // Download data
 
-        session.dataTask(with: url, completionHandler: { (data, response, error) in
+        session.dataTask(with: url) { (data, response, error) in
```

Comment:
I saw the examples but I didn't see any explicit rule so kept it like this.
Would you remove those? I wonder if it helps group the context and makes it easier to read when there is more than one. What do you think? Remove?